stacks-common/src/util/log.rs

```diff
@@ -258,9 +257,7 @@ fn inner_get_loglevel() -> slog::Level {
     }
 }
 
-lazy_static! {
-    static ref LOGLEVEL: slog::Level = inner_get_loglevel();
-}
+static LOGLEVEL: LazyLock<slog::Level> = LazyLock::new(|| inner_get_loglevel());
```

Comment:
Can probably simplify to

```suggestion
static LOGLEVEL: LazyLock<slog::Level> = LazyLock::new(inner_get_loglevel);
```